### DISCUSSION TITLE
Netbox-Docker update to 3.3.0

### DISCUSSION THREAD

**Discussion by enriquevm51** [netbox-community/netbox-docker#833] [General]
Hi, My docker instance of Netbox show me the message: 

New Release Available
NetBox v3.3.0 is available.

But the docker repository still in v3.2.9 (ebda466 27 days ago)

Are there any way to hide the message until this update could be downloaded?

Thanks in advance.

**Comment by tobiasge** [comment]
You can set `RELEASE_CHECK_URL` to `None` then Netbox will not check for new releases.

**Reply by enriquevm51** [reply]
Thank you.

I set it as

`RELEASE_CHECK_URL=`

and it is working for me.

**Reply by jonare77** [reply]
Did you set this in `netbox.env` ? Im still getting the message.

**Reply by enriquevm51** [reply]
I modified the file:
./env/netbox.env
Next commands:
docker-compose down
docker-compose up -d

**Comment by tionu** [comment]
I've set RELEASE_CHECK_URL to None in a seperate config-py:

`file: configuration/release_check.py`
```
# Disable Release Check
RELEASE_CHECK_URL = None
```

Comment: Previously I've set the RELEASE_CHECK_URL to the netbox-docker repository (https://api.github.com/repos/netbox-community/netbox-docker/releases), but this won't trigger an update alert because the version is compared to the genuine netbox version instead of netbox-docker version (https://github.com/netbox-community/netbox/blob/21b9732f061b7e91fa77c10da66bd098510354fc/netbox/netbox/views/__init__.py#L135).
